04-networking.md

```diff
@@ -91,8 +92,11 @@ The following two resource groups will be created and populated with networking
 
    > :book: Now that their regional hub has its first spoke, the hub can no longer run off of the generic hub template. The networking team creates a named hub template (e.g. `hub-eastus2.bicep`) to forever represent this specific hub and the features this specific hub needs in order to support its spokes' requirements. As new spokes are attached and new requirements arise for the regional hub, they will be added to this template file.
 
+   > :warning: If your cluster will be deployed in a region which has not yet konnectivity rolled out for communication from worker to master nodes, you might need to umcomment the firewall rule `pod-to-api-server_udp-1194` in `hub-regionA.bicep` before deployment.
```

Comment:
My only problem with this is that the user isn't going to know at this point, and this will cause anxiety (especially calling this a WARNING -- eek, I'd be nervous if I was going through this the first time -- in a panic because I have no way of checking). I think it might make sense to move this note to one of the "try it" pages after the cluster is deployed. Because you should be able to get through all the bicep deployments without error regardless of this.

First place you really have a hint that it might not be right would be on https://github.com/mspnp/aks-baseline/blob/main/07-bootstrap-validation.md. Can you find a spot there instead? That way if the reader gets through the instructions there, they will be fine, but if they get hung up, you'll have a hint for them.

Also, it doesn't seem to be JUST regional -- I deploy to eastus2 mostly, and haven't had this problem in about three years months. -- So maybe soften up the "regional" aspect of it?